### PR TITLE
Rings

### PR DESCRIPTION
**Pull request by amanr-dev**
Added my Animated Rings.

### CODE REVIEW

**Comment by amanr-dev** [comment]
Resolved MERGE CONFLICT in include.js.

**Comment by sophiabrandt** [comment]
@amanr-dev you are changing more than 5,000 lines in your pull request.

We cannot accept these changes, please make sure to [sync your fork with upstream](https://short.pm/forksync) and to apply _only_ the minimum necessary changes to `include.js`.

File: include.js [hunk 1]
```diff
@@ -69,34 +69,6 @@ let cards = [
...
-    author: 'Samriddhi',
-    githubLink: 'https://github.com/NarayanSam'
-  },
-  {
-    artName: 'RGB Square',
-    pageLink: './Art/TCrypt/animated.html',
-    imageLink: './Art/TCrypt/rgbsq-animated.gif',
-    author: 'T-Crypt',
-    githubLink: 'https://github.com/T-Crypt'
-  },
```
Comment (sophiabrandt) [inline include.js:99]: Is there a reason why you are removing these entries?

Comment (amanr-dev) [reply include.js:99]: No ma'am, I was trying to fix the issue of  "fix conflicts"

Comment (sophiabrandt) [reply include.js:99]: I understand. Please fix this issue so that you don't delete any valid entries in the `include.js` file.

Comment (amanr-dev) [reply include.js]: I added removed entries

Comment (sophiabrandt) [reply include.js]: @amanr-dev there are now any changes to `include.js`, so your entry for this PR is also missing.

**Pull request closed without merging**